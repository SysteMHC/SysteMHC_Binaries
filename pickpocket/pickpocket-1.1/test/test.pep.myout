# /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64/bin/pickpocket_pmbec -p ./pickpocket/pickpocket-1.1/test/test.pep -d /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64/data/PMBEC.lib
# Mon Jan  4 10:56:18 2016
# User: witold
# PWD : /home/witold/prog/SysteMHC_Binaries
# Host: Linux witold-VirtualBox 3.19.0-39-generic x86_64
# -p       1                    Use peptide input
# -d       /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64/data/PMBEC.lib Matrix library definitions
# Command line parameters set to:
#	[-rdir filename]     /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64 Home directory for NetMHpan
#	[-v]                 0                    Verbose mode
#	[-dirty]             0                    Dirty mode, leave tmp dir+files
#	[-tdir filename]     /home/witold/prog/temp Temporary directory (Default $$)
#	[-hlapseudo filename] /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64/data/MHC_pseudo.dat File with HLA pseudo sequences
#	[-hlaseq filename]                        File with full length HLA sequences
#	[-a line]            HLA-A02:01           HLA allele
#	[-f filename]                             File name with input
#	[-w]                 0                    w option for webface
#	[-s]                 0                    Sort output on descending affinity
#	[-p]                 1                    Use peptide input
#	[-l int]             9                    Peptide length [8-11]
#	[-inptype int]       0                    Input type [0] FASTA [1] Peptide
#	[-listMHC]           0                    Print list of alleles included in PickPocket
#	[-c filename]        /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64/data/contact.txt Contacts
#	[-d filename]        /home/witold/prog/SysteMHC_Binaries/pickpocket/pickpocket-1.1/Linux_x86_64/data/PMBEC.lib Matrix library definitions
#	[-pow float]         10.000000            Power for scoring function
# Input is in PEPTIDE format

# Peptide length 9
-----------------------------------------------------------------------------------
  pos          HLA    peptide        Identity 1-log50k(aff)  Exp_Binding
-----------------------------------------------------------------------------------
    0  HLA-A*02:01  AAAWYLWEV         PEPLIST         0.583        0.743
    0  HLA-A*02:01  AAGLQDCTM         PEPLIST         0.114        0.000
    0  HLA-A*02:01  AARNIVRRA         PEPLIST         0.066        0.000
    0  HLA-A*02:01  AARPDDPTL         PEPLIST         0.163        0.149
    0  HLA-A*02:01  AASCGGAVF         PEPLIST         0.017        0.149
    0  HLA-A*02:01  AASKQQMLM         PEPLIST         0.118        0.000
    0  HLA-A*02:01  AASSTHRKV         PEPLIST         0.237        0.272
    0  HLA-A*02:01  AEALLADGL         PEPLIST         0.202        0.085
    0  HLA-A*02:01  AEESLSLEA         PEPLIST         0.110        0.000
    0  HLA-A*02:01  AEFGPWQTV         PEPLIST         0.374        0.316
-----------------------------------------------------------------------------------
