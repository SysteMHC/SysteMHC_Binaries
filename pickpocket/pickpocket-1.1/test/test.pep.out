# /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64/bin/pickpocket_pmbec -p test.pep -d /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64/data/PMBEC.lib
# Fri Sep  5 21:18:27 2014
# User: mniel
# PWD : /usr/cbs/bio/src/pickpocket-1.1/test
# Host: Linux service0 2.6.32.12-0.7-default x86_64
# -p       1                    Use peptide input
# -d       /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64/data/PMBEC.lib Matrix library definitions
# Command line parameters set to:
#	[-rdir filename]     /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64 Home directory for NetMHpan
#	[-v]                 0                    Verbose mode
#	[-dirty]             0                    Dirty mode, leave tmp dir+files
#	[-tdir filename]     /scratch             Temporary directory (Default $$)
#	[-hlapseudo filename] /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64/data/MHC_pseudo.dat File with HLA pseudo sequences
#	[-hlaseq filename]                        File with full length HLA sequences
#	[-a line]            HLA-A02:01           HLA allele
#	[-f filename]                             File name with input
#	[-w]                 0                    w option for webface
#	[-s]                 0                    Sort output on descending affinity
#	[-p]                 1                    Use peptide input
#	[-l int]             9                    Peptide length [8-11]
#	[-inptype int]       0                    Input type [0] FASTA [1] Peptide
#	[-listMHC]           0                    Print list of alleles included in PickPocket
#	[-c filename]        /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64/data/contact.txt Contacts
#	[-d filename]        /usr/cbs/bio/src/pickpocket-1.1/Linux_x86_64/data/PMBEC.lib Matrix library definitions
#	[-pow float]         10.000000            Power for scoring function
# Input is in PEPTIDE format

# Peptide length 9
-----------------------------------------------------------------------------------
  pos          HLA    peptide        Identity 1-log50k(aff)  Exp_Binding
-----------------------------------------------------------------------------------
    0  HLA-A*02:01  AAAWYLWEV         PEPLIST         0.583        0.743
    0  HLA-A*02:01  AAGLQDCTM         PEPLIST         0.114        0.000
    0  HLA-A*02:01  AARNIVRRA         PEPLIST         0.066        0.000
    0  HLA-A*02:01  AARPDDPTL         PEPLIST         0.163        0.149
    0  HLA-A*02:01  AASCGGAVF         PEPLIST         0.017        0.149
    0  HLA-A*02:01  AASKQQMLM         PEPLIST         0.118        0.000
    0  HLA-A*02:01  AASSTHRKV         PEPLIST         0.237        0.272
    0  HLA-A*02:01  AEALLADGL         PEPLIST         0.202        0.085
    0  HLA-A*02:01  AEESLSLEA         PEPLIST         0.110        0.000
    0  HLA-A*02:01  AEFGPWQTV         PEPLIST         0.374        0.316
-----------------------------------------------------------------------------------
